firefox_ui_tests/remote/security/test_mixed_script_content_blocking.py

```diff
@@ -0,0 +1,100 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestMixedScriptContentBlocking(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.test_url = "https://mozqa.com/data/firefox/security/mixed_content_blocked/index.html"
+        self.test_elements = [
+            ('result1', 'Insecure script one'),
+            ('result2', 'Insecure script from iFrame'),
+            ('result3', 'Insecure plugin'),
+            ('result4', 'Insecure stylesheet'),
+        ]
+        self.locationbar = self.browser.navbar.locationbar
+
+    def tearDown(self):
+        self.marionette.set_context("chrome")
```

Comment:
This fixes the failure in teardown that occurs because marionette leaves us in content scope when a test is skipped. Hiding this popup should never throw, if it does we want the test to fail, which wouldn't happen if we use try/finally.